### PR TITLE
Feat: add extra fields to "create user" (/admin/user) endpoint

### PR DESCRIPTION
**Pull request by pieterlukasse**
Link to JIRA ticket if there is one: https://ctds-planx.atlassian.net/browse/VADC-1350

### Improvements
- feat: add extra (optional) fields to "create user" `/admin/user` POST endpoint 
 to make user creation more uniform, reflecting what is also done elsewhere
 in [fence/sync/sync_users.py _upsert_userinfo()](https://github.com/uc-cdis/fence/blob/eb90523fc544a7704a2e5c1cbc9be592fb280bfc/fence/sync/sync_users.py#L1119) for example

### CODE REVIEW

File: tests/admin/test_admin_users.py [hunk 1]
```diff
@@ -30,6 +30,35 @@ def test_create_user(db_session, oauth_client):
     assert user.username == "insert_user"
     assert user.is_admin == True
     assert user.email == "insert_user@fake.com"
+    assert user.display_name is None
+    assert user.phone_number is None
+    assert user.identity_provider is None
+    assert len(user.tags) == 0
+
+
+def test_create_user_with_all_fields_set(db_session, oauth_client):
```
Comment (Avantol13) [inline tests/admin/test_admin_users.py:39]: quick docstring describing the test, please

Comment (pieterlukasse) [reply tests/admin/test_admin_users.py:39]: resolved

**Review by paulineribeyre — DISMISSED** [review]
lgtm

**Review by paulineribeyre — APPROVED** [review]

**Comment by paulineribeyre** [comment]
The bullet points in this PR description are not displaying well in the release notes, please read https://github.com/uc-cdis/cdis-wiki/blob/master/dev/release_notes.md for guidance

**Pull request merged**